src/api/queries/getScheduledItems.ts

```diff
@@ -9,18 +9,9 @@ export const getScheduledItems = gql`
       totalCount
       scheduledDate
       items {
-        externalId
-        createdAt
-        createdBy
-        updatedAt
-        updatedBy
-        scheduledDate
-        scheduledSurfaceGuid
-        approvedItem {
-          ...CuratedItemData
-        }
+        ...ScheduledItemData
       }
     }
   }
-  ${CuratedItemData}
+  ${ScheduledItemData}
 `;
```

Comment:
Updated to use the `ScheduledItemData` which had the `source` field added in the parent ticket. I _think_, initially, this query didn't pull in all the fields from scheduled items, which is why fields were listed individually instead of using the fragment with all the available fields.